src/Libraries/CoreNodeModelsWpf/Properties/CoreNodeModelWpfResources.resx

```diff
@@ -123,36 +123,171 @@
+List&lt;var&gt;</value>
+  </data>
+  <data name="ChartsBarChartColorsDataPortToolTip" xml:space="preserve">
+    <value>A list of colors for each bar chart category.
+
+List&lt;color&gt;</value>
+  </data>
+  <data name="ChartsBarChartLabelsValuesDataPortToolTip" xml:space="preserve">
+    <value>Dictionary containing label:value key-pairs
+
+Dictionary&lt;string, double&gt;</value>
+  </data>
   <data name="ChartsBasicLineChartDescription" xml:space="preserve">
     <value>Create a new Basic Line Chart</value>
   </data>
   <data name="ChartsBasicLineChartSearchTags" xml:space="preserve">
     <value>CoreNodeModelsWpf.Charts.Index-ValueLinePlot;Index-Value Line Plot;indexvaluelineplot</value>
   </data>
+  <data name="ChartsBasicLineChartLabelsDataPortToolTip" xml:space="preserve">
+    <value>A list of string labels for each line to be plotted.
+
+List&lt;string&gt;</value>
+  </data>
+  <data name="ChartsBasicLineChartValuesDataPortToolTip" xml:space="preserve">
+    <value>List of lists each containing double values to be plotted against X-Axis values.
+
+List&lt;List&lt;double&gt;&gt;</value>
+  </data>
+  <data name="ChartsBasicLineChartColorsDataPortToolTip" xml:space="preserve">
+    <value>A list of colors for each line.
+
+List&lt;color&gt;</value>
+  </data>
+  <data name="ChartsBasicLineChartLabelsValuesDataPortToolTip" xml:space="preserve">
+    <value>Dictionary containing label:value key-pairs
+
+Dictionary&lt;string, List&lt;double&gt;&gt;</value>
+  </data>
   <data name="ChartsHeatSeriesDescription" xml:space="preserve">
     <value>Create a heat series plot</value>
   </data>
   <data name="ChartsHeatSeriesSearchTags" xml:space="preserve">
     <value>CoreNodeModelsWpf.Charts.HeatSeriesPlot;Heat Series Plot;heatseriesplot</value>
   </data>
+  <data name="ChartsHeatSeriesXLabelsDataPortToolTip" xml:space="preserve">
+    <value>A list of string labels for the x-axis.
+
+List&lt;string&gt;</value>
+  </data>
+  <data name="ChartsHeatSeriesYLabelsDataPortToolTip" xml:space="preserve">
+    <value>A list of string labels for the y-axis.
+
+List&lt;string&gt;</value>
+  </data>
+  <data name="ChartsHeatSeriesValuesDataPortToolTip" xml:space="preserve">
+    <value>A list of lists each containing double values representing items in a column. 
+
+List&lt;List&lt;double&gt;&gt;</value>
+  </data>
+  <data name="ChartsHeatSeriesColorsDataPortToolTip" xml:space="preserve">
+    <value>A list of colors for each bar chart category.
+
+List&lt;color&gt;</value>
+  </data>
+  <data name="ChartsHeatSeriesLabelsValuesDataPortToolTip" xml:space="preserve">
+    <value>Dictionary containing label:value key-pairs
+
+Dictionary&lt;string, Dictionary&lt;string, double&gt;&gt;</value>
+  </data>
   <data name="ChartsPieChartDescription" xml:space="preserve">
     <value>Create a new Pie Chart</value>
   </data>
   <data name="ChartsPieChartSearchTags" xml:space="preserve">
     <value>CoreNodeModelsWpf.Charts.PieChart;Pie Chart; piechart</value>
```

Comment:
Should the space before 'piechart' should be removed here as well like the other resources file?

Reply:
This is picked up in this [PR](https://github.com/DynamoDS/Dynamo/pull/13843).

Reply:
Fixed in this https://github.com/DynamoDS/Dynamo/pull/13843